arviz/plots/backends/bokeh/elpdplot.py

```diff
@@ -163,9 +163,7 @@ def _plot_atomic_elpd(
     marker = _validate_bokeh_marker(plot_kwargs.get("marker"))
     marker_func = getattr(mk, marker)
     sizes = np.ones(len(xdata)) * plot_kwargs.get("s")
-    glyph = marker_func(
-        x="xdata", y="ydata", size="sizes", line_color=plot_kwargs.get("color", "black")
-    )
+    glyph = Scatter(x="xdata", y="ydata", size="sizes", line_color=plot_kwargs.get("color", "black"), marker='cross')
```

Comment:
note that while the other instances call `Dash` explicitly, here the `marker_func` is used instead, which allows users to choose the marker to be used. I believe the suggestion will fix this change in funcitonality, marker is already defined correctly, but you should definitely try it out locally

```suggestion
    glyph = Scatter(x="xdata", y="ydata", size="sizes", line_color=plot_kwargs.get("color", "black"), marker=marker)
```